mos-platform/common/lib/exception.cc

```diff
@@ -0,0 +1,24 @@
+#include <exception>
+#include <stdlib.h>
+
+namespace std {
+    // __current_terminate_handler shall be never-null.
+    // It's either abort() or the function provided at set_terminate.
+    static terminate_handler __current_terminate_handler = ::abort;
+
+    // Current standard explicilty says the default handler value does not have to be nullptr.
+    terminate_handler get_terminate() noexcept {
```

Comment:
Add a note that these aren't thread-safe in the presence of user thread implementations.

Reply:
Actually, if you would, do this in the header, not the implementation. More likely to be seen that way.

Reply:
I've created llvm-mos/llvm-mos#138 as a potentially more principled solution for atomicity and thread safety in llvm-mos.

Reply:
Actually, forget the whole thing. We should have this as another callout of things not yet supported by the compiler/SDK/runtime, like exceptions. No action required.